src/main/java/com/recruitment/project/parking/parkingspaces/domain/UserParkingVisit.java

```diff
@@ -0,0 +1,26 @@
+package com.recruitment.project.parking.parkingspaces.domain;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.Data;
+import net.minidev.json.annotate.JsonIgnore;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@Table(name = "UserParkingVisit")
+public class UserParkingVisit {
+
+    @Id
```

Comment:
Nieczytelnie mocno tak ciurkiem zapisane.

Reply:
Tutaj chodzi o dodanie dodatkowego entera pomiędzy atrybutami klasy?

Reply:
Tak, jak masz adnotacje nad polami to to nieczytelnie wygląda.

Reply:
Dzięki, zapamiętam.